packages/phrases/src/locales/ja/errors/hook.ts

```diff
@@ -1,6 +1,7 @@
 const hook = {
   missing_events: '少なくとも1つのイベントを提供する必要があります。',
   send_test_payload_failed: 'テストペイロードの送信に失敗しました：{{message}}',
+  maximum_webhook_limit_exceeded: '最大限度 {{value, number}} のWebフックが達成されました。',
```

Comment:
```suggestion
  maximum_webhook_limit_exceeded: '最大限度 {{value, number}} の Web フックが達成されました。',
```

Reply:
Can we update the prompt to auto add spaces?

Reply:
Good call, do it in the cool-down cycle. LOG-6212